homeassistant/config_entries.py

```diff
@@ -3420,6 +3420,11 @@ def _entry_id(self) -> str:
         """Return config entry id."""
         return self.handler[0]
 
+    @property
+    def _subentry_type(self) -> str:
+        """Return type of subentry we are editing/creating."""
+        return self.handler[1]
```

Comment:
I'm missing a test addition in `test_config_entries.py` for this addition.

Reply:
This is really not nice.
Why does a PR titled "Fixes in Google AI TTS" add new untested and undocumented functionality to `homeassistant/config_entries.py`?

Reply:
https://github.com/home-assistant/core/pull/147565

Reply:
I copied this function from Paulus' https://github.com/home-assistant/core/pull/146766 which also didn't have a test. Adding tests in https://github.com/home-assistant/core/pull/147566